locale/en/get-involved/index.md

```diff
@@ -9,7 +9,6 @@ layout: contribute.hbs
 
 * The [GitHub issues list](https://github.com/nodejs/node/issues) is the place for discussion of Node.js core features.
 * For real-time chat about Node.js development use one of the platforms below
-  * For IRC, go to `irc.freenode.net` in the `#node.js` channel with an [IRC client](https://en.wikipedia.org/wiki/Comparison_of_Internet_Relay_Chat_clients) or connect in your web browser to the channel using [freenode's WebChat](https://webchat.freenode.net/#node.js).
```

Comment:
we could replace it with libera.chat in the meantime - it's safely owned and moderated.

Reply:
Are we actually moving to libera officially?

Reply:
I think we should; either way, the channel exists and is moderated by the same folks that ran the freenode one.